tests/test_logger.py

```diff
@@ -49,3 +52,20 @@ class A(object):
         args.verbose = True
         CmdBase._set_loglevel(args)
         self.assertEqual(Logger.logger().getEffectiveLevel(), logging.DEBUG)
+
+    @patch('sys.stderr.write')
+    @patch('sys.stdout.write')
+    def test_stderr(self, mock_stdout, mock_stderr):
+        ret = main(['config', 'non_existing_section.field'])
```

Comment:
So sorry for bothering, I didn't think this through right away :slightly_frowning_face: . There is no reason to use `main()` here actually, you could just use `Logger.error('some msg')` and then check if it is what was written to stderr. Same with Logger.info and stdout. It would be even better if you could check that the message written is the one you expect, since it would be great to be covered from that side as well from potential problems.

Reply:
Right, it is surely easier. I'm on it. :)

Reply:
Thank you!

Reply:
One question though: dvc logger, on error, attaches 'Error' string and chat. Is it okay if I will just do partial check of string?

Reply:
Sure, something like `self.assertEqual(msg, "Error: mymsg")` should be enough.